includes/Rest/AnyCommentRestComment.php

```diff
@@ -579,7 +579,7 @@ public function create_item ( $request ) {
 		$should_use_wordpress_login_form = ! is_user_logged_in() && AnyCommentGenericSettings::is_form_type_wordpress();
 
 		if ( $should_use_wordpress_login_form ) {
-			return new WP_Error( 'rest_use_wordpress_to_login', __( 'Please login to leave a comment', 'anycomment' ), [ 'status' => 400 ] );
+			return new WP_Error( 'rest_use_wordpress_to_login', __( 'Please login or register to leave a comment', 'anycomment' ), [ 'status' => 400 ] );
```

Comment:
Not sure if this is very valid, because users do not see registration link anywhere. Thoughts?

Reply:
If WordPress Only is selected as the group who should be allowed to comment, this message is displayed. If the user does not already have an account perhaps they should be steered toward creating one?